api/src/pcapi/core/educational/commands.py

```diff
@@ -90,7 +90,7 @@ def import_deposit_csv(path: str, year: int, ministry: str, conflict: str, final
             print("\033[91mERROR: UAICode or depositAmount missing in CSV headers\033[0m")
             return
         data: dict[str, Decimal] = {}
-        # sometime we get 1 row per institution and somtime 1 row per class.
+        # sometime we get 1 row per institution and sometime 1 row per class.
```

Comment:
```suggestion
        # sometime we get 1 row per institution and sometimes 1 row per class.
```

Reply:
@dbaty Je double la mise xD

```suggestion
        # sometimes we get 1 row per institution and sometimes 1 row per class.
```

Reply:
J'étais tellement concentré sur le mot corrigé que je n'ai pas vu la première occurrence de "sometime". :) Bien vu !